opentelemetry/src/main/scala/zio/telemetry/opentelemetry/Tracing.scala

```diff
@@ -227,13 +235,14 @@ object Tracing {
       def createChildOf(parent: Span, spanName: String, spanKind: Span.Kind): UManaged[Span] =
         for {
           nanoSeconds <- currentNanos.toManaged_
+          context     = parent.storeInContext(Context.root())
```

Comment:
`Span.storeInContext` is just a wrapper for `Context.with`:

```
default Context storeInContext(Context context) {
    return context.with(SpanContextKey.KEY, this);
  }
```

Reply:
Hence the "nit". ;-) I just checked the docs because I found the direction a bit unusual and it's in fact recommended against. No clue why they added 'storeInContext' if they recommend using 'with'. Anyway, feel free to change or keep this as is.